bonita/content.md

```diff
@@ -14,397 +14,281 @@ Bonita (called Bonita BPM till 7.5) is an open-source business process managemen
+## Start Bonita with custom security credentials
 
-Important note: There are several ways to store data used by applications that run in Docker containers. We encourage users of the `%%REPO%%` images to familiarize themselves with the options available, including:
+```
+docker run --name=bonita -h <hostname> -e "TENANT_LOGIN=tech_user" -e "TENANT_PASSWORD=secret" -e "PLATFORM_LOGIN=pfadmin" -e "PLATFORM_PASSWORD=pfsecret" -d -p 8080:8080 %%IMAGE%%
+```
 
--	Let Docker manage the storage of your data [by writing the files to disk on the host system using its own internal volume management](https://docs.docker.com/engine/tutorials/dockervolumes/#adding-a-data-volume). This is the default and is easy and fairly transparent to the user. The downside is that the files may be hard to locate for tools and applications that run directly on the host system, i.e. outside containers.
--	Create a data directory on the host system (outside the container) and [mount this to a directory visible from inside the container](https://docs.docker.com/engine/tutorials/dockervolumes/#mount-a-host-directory-as-a-data-volume). This places the database files in a known location on the host system, and makes it easy for tools and applications on the host system to access the files. The downside is that the user needs to make sure that the directory exists, and that directory permissions and other security mechanisms on the host system are set up correctly.
+Now you can access the Bonita Runtime on localhost:8080/bonita and login using: tech_user / secret
 
-The Docker documentation is a good starting point for understanding the different storage options and variations, and there are multiple blogs and forum postings that discuss and give advice in this area. We will simply show the basic procedure here for the latter option above:
+## Where data is stored
 
-1.	Create a data directory on a suitable volume on your host system, e.g. `/my/own/datadir`.
-2.	Start your `%%REPO%%` container like this:
+Bonita uses tomcat that writes file to a working directory and a temp directory.
 
-	docker run --name some-%%REPO%% -v /my/own/datadir:/opt/bonita -d -p 8080:8080 %%IMAGE%%:tag
+It can be a good practice to mount the following folders into volumes
 
-The `-v /my/own/datadir:/opt/bonita` part of the command mounts the `/my/own/datadir` directory from the underlying host system as `/opt/bonita` inside the container, where Bonita will deploy the bundle and write data files by default.
+* `/opt/bonita/server/temp`
+* `/opt/bonita/server/work`
 
-## Migrate from an earlier version of Bonita
+## Environment variables
 
--	Stop the container to perform a backup
+When you start the bonita image, you can adjust the configuration of the Bonita instance by passing one or more environment variables on the docker run command line.
 
-	```console
-	$ docker stop bonita_7.9.5_postgres
-	```
+### PLATFORM_PASSWORD
```

Comment:
I would put login before password, for PLATFORM, TENANT, like it is for MONITORING.